Week2/akif.js

```diff
@@ -0,0 +1,108 @@
+// Part 1 Hello World
+console.log('Hello World'); // English
+console.log('Merhaba Dunya'); // Turkish
+console.log('Hallo Wereld'); // Dutch
+console.log('مرحبا بالعالم'); // Arabic
+
+// Part 2 Error  -----------------------------------------
+console.log("I'm awesome");
```

Comment:
Just for your information, you can also use escape characters(using backslash) like this:
`console.log('I\'m awesome');`

The middle single quote will be then interpreted by JS as a string character and not a special character.